base/applications/games/spider/spider.h

```diff
@@ -2,6 +2,7 @@
 #define _SPIDER_PCH_
 
 #include <cardlib.h>
+#include <shellapi.h>
```

Comment:
Maybe include the shellapi.h only in the source file that really needs it (here: spider.cpp, and for solitaire: solitaire.cpp)

Reply:
Moved.